consensus/ethash/ethash.go

```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/ethereum/go-ethereum/consensus"
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/rpc"
-	metrics "github.com/rcrowley/go-metrics"
+	metrics "github.com/nonsense/go-metrics"
```

Comment:
Should we fork to `ethereum` or `ethersphere` instead so the team can manage this dependency? I've just hit this issue in the past where a user's fork needs to be forked again to make changes.

Reply:
Good point. I'm happy with any repo. `ethersphere` or `ethereum` would both work. If we don't want to keep `go-metrics` and `go-metrics-influxdb` on `ethereum`, I would be happy to fork them on `ethersphere` so that at least there are more people who have access to it.

Reply:
Actually my first thought was to vendor it like the `log` package which you pointed out, but there is already a `metrics` package which does other things, and having the `go-metrics` under the `metrics` folder seemed weird. But maybe we can vendor all the metrics related libraries under `metrics/lib` in go-ethereum?

Reply:
Why do we need to fork go-metrics? Can't we just add one more metric of our own (`ResettingTimer`)?

Reply:
We would also have to create our own `Registry` (see https://github.com/ethereum/go-ethereum/pull/15910/files#diff-69f4fcff395026c7300bbaa7c953e251R199), when the `StandardRegistry` is good for our purposes.

Reply:
In my opinion `metrics` library is something that doesn't change often, so I'd rather keep all related code in one library, rather than patch additional functionality with external structs. Because of this I found forking to be the easiest. I'm happy to hear other opinions on this though.

Reply:
@lmars changed fork location to `ethersphere`.